text/0000-format-args-implicit-identifiers.md

```diff
@@ -163,17 +246,30 @@ For a comparison of the three languages, the following code would be the equival
     // Javascript
     `${greeting} ${person}`
 
-It is the RFC author's experience that Python and Javascript's functionality read easily from left-to-right and it is clear where each variable is being substituted into the format string.
+    // C# / VB
+    $"{greeting} {person}"
+
+    // Swift
+    "\(greeting) \(person)"
+
+    // Ruby
+    "#{greeting} #{person}"
+
+    // Scala / PHP
+    s"$greeting $person"
```

Comment:
The prefix `s` is Scala-specific. PHP doesn't need the `s`.

Reply:
Thanks, I merged them for conciseness and then got it wrong 🤦‍♂